curator/README.md

```diff
@@ -103,26 +99,104 @@ months from that date.
 If you want to be exact with curator, it is best to use `days` e.g. `delete: days: 31`
 [Curator issue](https://github.com/elastic/curator/issues/569)
 
+### Compatibility with OpenShift 3.7
+In earlier releases admins could control the timezone and time when curator runs from the curator config. This configuration has been moved to openshift-ansible. The [timezone is not currently configurable](https://github.com/kubernetes/kubernetes/issues/47202), instead the timezone of OpenShift master node is used.
+
+### Using actions file
+OpenShift custom config file format ensures that important internal indices don't get deleted by mistake. In order to use the actions file add an [exclude](https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/fe_exclude.html) rule to your configuration to retain these indices. You also need to manually add all the other patterns, see action 3 in the below example:
+```
+actions:
+  1:
+    action: delete_indices
+    description: be careful!
+    filters:
+    - exclude: false
+      kind: regex
+      filtertype: pattern
+      value: '^project\.myapp\..*$'
+    - direction: older
+      filtertype: age
+      source: name
+      timestring: '%Y.%m.%d'
+      unit_count: 7
+      unit: days
+    options:
+      continue_if_exception: false
+      timeout_override: '300'
+      ignore_empty_list: true
+  2:
+    action: delete_indices
+    description: be careful!
+    filters:
+    - exclude: false
+      kind: regex
+      filtertype: pattern
+      value: '^\.operations\..*$'
+    - direction: older
+      filtertype: age
+      source: name
+      timestring: '%Y.%m.%d'
+      unit_count: 56
+      unit: days
+    options:
+      continue_if_exception: false
+      timeout_override: '300'
+      ignore_empty_list: true
+  3:
+    action: delete_indices
+    description: be careful!
+    filters:
+    - exclude: true
+      kind: regex
+      filtertype: pattern
+      value: '^project\.myapp\..*$|^\.operations\..*$|^\.searchguard\..*$|^\.kibana$'
+    - direction: older
+      filtertype: age
+      source: name
+      timestring: '%Y.%m.%d'
+      unit_count: 30
+      unit: days
+    options:
+      continue_if_exception: false
+      timeout_override: '300'
+      ignore_empty_list: true
+```
+
+### Modifying configuration
 To create the curator configuration, you can just edit the current
 configuration in the deployed configmap:
 
     $ oc edit configmap/logging-curator
 
-If this does not redeploy automatically, redeploy manually:
-
-    $ oc deploy --latest logging-curator
-
-For scripted deployments, do this:
-
-    $ create /path/to/mycuratorconfig.yaml
-    $ oc delete configmap logging-curator ; sleep 1
-    $ oc create configmap logging-curator --from-file=config.yaml=/path/to/mycuratorconfig.yaml ; sleep 1
+For scripted deployments, copy the configuration file that was created by the installer and create your new OpenShift custom config:
+```
+$ create /path/to/configuration.yaml
```

Comment:
A lot of times we have people copy and paste code blocks like this... could you update this so they have a command they can just run to copy out the `configuration.yaml` block? likely something with oc get and an output redirect to `/path/to/configuration/yaml`